packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -1,27 +1,179 @@
 import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
-// import { SliderState } from './Slider.types';
+import { createFocusIndicatorStyleRule } from '@fluentui/react-tabster';
+import { SliderState } from './Slider.types';
 
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
+const useRootStyles = makeStyles({
   root: theme => ({
-    // TODO Add default styles for the root element
+    '--slider-thumb-size': '20px',
+    '--slider-color': '#005fb8',
+    position: 'relative',
+    height: 'var(--slider-thumb-size)',
+    width: '280px',
```

Comment:
I *think* (not 100% sure) this width should probably be `minWidth`; otherwise if the container is set to `flex`, it won't grow automatically.

Reply:
Sometimes the component will live in a flex or a grid, where the container provides the constraints. When this happens you want it to fill the container. So either we have a `fluid` property like before for this case, or maybe consider what the right practice is here.